src/stores/order-store-base.js

```diff
@@ -9,7 +9,7 @@ class OrderStoreBase extends StoreBase {
   async getNextOrder() {
     const query = { _id: this.orderKey };
     const update = { $inc: { seq: 1 } };
-    const options = { upsert: true, returnOriginal: false };
+    const options = { upsert: true, returnDocument: 'after' };
```

Comment:
I guess this is a breaking change in the driver...

Reply:
No, it's just a different way to configure the same thing. The behavior won't change.